### ISSUE TITLE
Video autoplays

### ISSUE THREAD

**Issue by JenM00re** [Swarthmore/a11y-doc-repo#15]
Accessibility issue with videos: should not / cannot autoplay I think this is the documentation: [Understanding Success Criterion 2.3.3: Animation from Interactions | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)]

Tony suggested the media css prefers reduced motion. 

So, it'll be similar to the code from the other issue:
```
@media (prefers-color-scheme: dark) {
body { /* Light text on dark background */
color: #ffffff;
background-color: #222222;
}
}
```
